cmd/terradoc/cli/validate.go

```diff
@@ -80,17 +108,40 @@ func (vcm ValidateCmd) Run() error {
 	return nil
 }
 
-func printValidationSummary(summary validators.Summary, docFilename, defFilename string) {
+func printValidationSummary(summary validators.Summary, docFilename string) {
 	for _, missingDef := range summary.MissingDefinition {
-		fmt.Fprintf(os.Stderr, "Missing %s definition: %q is not defined in %q\n", summary.Type, missingDef, defFilename)
+		fmt.Fprintf(os.Stderr, "Missing %s definition: %q is not defined in .tf files\n", summary.Type, missingDef)
 	}
 
 	for _, missingDoc := range summary.MissingDocumentation {
 		fmt.Fprintf(os.Stderr, "Missing %s documentation: %q is not documented in %q\n", summary.Type, missingDoc, docFilename)
```

Comment:
docFilename should not be the absolute path but the name of the parameter given.

Reply:
@mariux Could you clarify this? What do you mean by the name of the parameter given?